bin/json2csv.js

```diff
@@ -35,6 +35,7 @@ program
   .option('-a, --include-empty-rows', 'Includes empty rows in the resulting CSV output.')
   .option('-b, --with-bom', 'Includes BOM character at the beginning of the csv.')
   .option('-p, --pretty', 'Use only when printing to console. Logs output in pretty tables.')
+  .option('-P, --pretty-without-color', 'Use only when printing to console. Logs output in pretty tables, but without color.')
```

Comment:
this is unnecessary. `cli-table3` through `colors` already supports a `--no-color` by default, so a dedicated option should not be needed here.